src/styles/blocks.scss/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
```

Comment:
Please pay attention to the CSS properties order
https://9elements.com/css-rule-order/